MIGRATING.md

```diff
@@ -0,0 +1,261 @@
+-        assert_eq!((42, pk1.to_vec()), marias[0].0);
+-        assert_eq!((24, pk3.to_vec()), marias[1].0);
++        assert_eq!(pk1, marias[0].0);
++        assert_eq!(pk3, marias[1].0);
+
+         // Data
+         assert_eq!(data1, marias[0].1);
+```
+
+- Remove the primary key from the `MultiIndex` key specification [\#533](https://github.com/CosmWasm/cw-plus/issues/533) /
+`MultiIndex` primary key spec removal [\#569](https://github.com/CosmWasm/cw-plus/pull/569)
+
+The primary key is no longer needed when specifying `MultiIndex` keys. This simplifies both `MultiIndex` definitions
+and usage.
+
+Required changes are along the lines of:
+```diff
+diff --git a/packages/storage-plus/src/indexed_map.rs b/packages/storage-plus/src/indexed_map.rs
+index 022a4504..c7a3bb9d 100644
+--- a/packages/storage-plus/src/indexed_map.rs
++++ b/packages/storage-plus/src/indexed_map.rs
+@@ -295,8 +295,8 @@ mod test {
+     }
+
+     struct DataIndexes<'a> {
+-        // Second arg is for storing pk
+-        pub name: MultiIndex<'a, (String, String), Data, String>,
++        // Last type parameters are for signaling pk deserialization
++        pub name: MultiIndex<'a, String, Data, String>,
+         pub age: UniqueIndex<'a, u32, Data, String>,
+         pub name_lastname: UniqueIndex<'a, (Vec<u8>, Vec<u8>), Data, String>,
+     }
+@@ -326,11 +326,7 @@ mod test {
+     // Can we make it easier to define this? (less wordy generic)
+     fn build_map<'a>() -> IndexedMap<'a, &'a str, Data, DataIndexes<'a>> {
+         let indexes = DataIndexes {
+-            name: MultiIndex::new(
+-                |d, k| (d.name.clone(), unsafe { String::from_utf8_unchecked(k) }),
+-                "data",
+-                "data__name",
+-            ),
++            name: MultiIndex::new(|d| d.name.clone(), "data", "data__name"),
+             age: UniqueIndex::new(|d| d.age, "data__age"),
+             name_lastname: UniqueIndex::new(
+                 |d| index_string_tuple(&d.name, &d.last_name),
+@@ -469,8 +462,8 @@ mod test {
+         // index_key() over MultiIndex works (empty pk)
+         // In a MultiIndex, an index key is composed by the index and the primary key.
+         // Primary key may be empty (so that to iterate over all elements that match just the index)
+-        let key = ("Maria".to_string(), "".to_string());
+-        // Use the index_key() helper to build the (raw) index key
++        let key = "Maria".to_string();
++        // Use the index_key() helper to build the (raw) index key with an empty pk
+         let key = map.idx.name.index_key(key);
+         // Iterate using a bound over the raw key
+         let count = map
+```
+
+- Incorrect I32Key Index Ordering [\#489](https://github.com/CosmWasm/cw-plus/issues/489) /
+Signed int keys order [\#582](https://github.com/CosmWasm/cw-plus/pull/582)
+
+As part of range iterators revamping, we fixed the order of signed integer keys. You shouldn't change anything in your
```

Comment:
This is super important. Can we add this to the top of this MIGRATING.md section?

It can be here as well, but one or two sentences about it on top, then the details below would be nice.

(I know it is merged, but this can go anytime, people read MIGRATING.md from main